### PR TITLE
Add some initial rules for discussion

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: linters/.eslintrc [hunk 1]
```diff
@@ -2,4 +2,33 @@
...
+    // http://eslint.org/docs/rules/comma-dangle
+    "comma-dangle": ["error", "only-multiline"],
+
+    // Enforce consistent spacing inside braces
+    //http://eslint.org/docs/rules/object-curly-spacing
+    "object-curly-spacing": [2, "always"],
+
+    // Require or allow JSX to close on the same line
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-closing-bracket-location.md
+    "jsx-closing-bracket-location": [1, "props-aligned"]
```
Comment (Ollo) [inline linters/.eslintrc]:
```
// 'jsx-closing-bracket-location': [1, 'props-aligned']

<Hello 
  firstName="John"
  lastName="Smith" />

<Say
  firstName="John"
  lastName="Smith">
  Hello
</Say>
```

ahhhhhh 😛

Comment (dmccraw) [inline linters/.eslintrc]: Hmm, not a fan of this. I prefer having it on the same indentation as the opening. Your second example makes it hard to know that `Hello` is not part of the props.

Comment (Ollo) [inline linters/.eslintrc]: I agree the readability suffers a bit with the second example.. I typically will indent the contained text in situations like that. The reason I added this is for situations where large composition of classNames (BEM) forces new lines and makes things really funky to read.

File: linters/.eslintrc [hunk 2]
```diff
@@ -2,4 +2,29 @@
 // Copy this file, and add rule overrides as needed.
 {
   "extends": "airbnb"
+  "rules": {
+    // Require or disallow semicolons instead of ASI
+    // http://eslint.org/docs/rules/semi
+    "semi" : [2, "never"],
+
```
Comment (amberwebb) [inline linters/.eslintrc]: I have been leaving semicolons out of my tests and I have to say it's kind of... freeing. So thumbs up on this and I retract my previous comments about preferring semicolons.

File: linters/.eslintrc (hunk 1 above)
Comment (amberwebb) [inline linters/.eslintrc]: I like the first one.

**Pull request merged**